compiler/resolution/functionResolution.cpp

```diff
@@ -6457,7 +6553,46 @@ resolveExpr(Expr* expr) {
         implementForallIntents2wrapper(call, eflopiHelper);
       }
 
-      resolveFns(call->isResolved());
+      // For ContextCallExprs, be sure to resolve all of the
+      // functions that could be called.
+      if (ContextCallExpr* cc = toContextCallExpr(call->parentExpr)) {
+        Type* firstRetType = NULL;
+        Type* retType = NULL;
+        bool same = true;
+        for_alist(optionExpr, cc->options) {
```

Comment:
This logic is shorter to write without the loop, given that there are exactly two options.
I am OK with it as-is, though.